### PR TITLE
Design UI with more styling

### PR DESCRIPTION
**Pull request by lawrahkonwea**
The look and feel of the application was improved by adding the styles of my choice with HTML, CSS and Javascript;
- Still kept the general layout of the wireframe.

### CODE REVIEW

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @lawrahkonwea , 

Good job so far! you have done well styling the application
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
Styling applied :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

- [ ] **HI @lawrahkonwea , Please notice that we always have to make sure that we have documented our work in a professional way, currently you have added the readme template correctly but no information is filled in and it is just an empty template right now, It should contain all the necessary details of our project, this will make it easier for anyone checking out this project and will also showcase your good documentation skill, which is also important, Please make sure to update your readme file with all the necessary information.**
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -207,7 +201,7 @@ Example:
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
+- GitHub: [@githubhandle](https://github.com/lawrahkonwea)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (rloterh) [inline README.md:207]: Kindly ensure to improve upon the following aspects of your ReadMe (which are required review criteria), like informed in the previous review:
- [ ] OPTIONAL
Kindly ensure to remove the template logo. Optionally, you can replace it with your own logo if you have one.
- [ ] OPTIONAL
Kindly ensure that the _Future Features_ section of your ReadMe includes 1-3 potential features to be added later.
- [ ] OPTIONAL
This project appears to be worked on by a sole contributor, therefore kindly update the Author section of the ReadMe, remove redundant contributor(s) details and change the header title _Authors_ to _Author_ accordingly. 
- [ ] OPTIONAL
Kindly endeavor to fill the FAQ section with at least 2 Q&A(s).
- [ ] OPTIONAL
 MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Amaka,  

Your project is complete! There is nothing else to say other than... it's time to merge it. Congratulations!  🎉 

## Highlights 👍
- Nice job making touches to your project!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (lawrahkonwea) [reply README.md:207]: Thank you

**Pull request merged**